client/src/views/files/handler/editor/FileEditor.vue

```diff
@@ -28,6 +91,11 @@ const workspaceHandleRef = ref<number | undefined>(undefined);
 const documentPathRef = ref<FsPath>();
 const cryptpadInstance = ref<Cryptpad | null>(null);
 const fileUrl = ref<string | null>(null);
+const error = ref('');
+
+function reloadPage(): void {
+  window.location.reload();
```

Comment:
`PARSEC_APP_TESTBED_AUTO_LOGIN=false`
`PARSEC_APP_ENABLE_ACCOUNT_AUTO_LOGIN=false`
 and then test it.